packages/libs/react-ui/src/components/Link/Link.css.ts

```diff
@@ -0,0 +1,32 @@
+import { sprinkles, vars } from '../../styles';
+
+import { style } from '@vanilla-extract/css';
+
+export const linkContainerClass = style([
+  sprinkles({
+    display: 'flex',
+  }),
+  {
+    selectors: {
+      '&:hover': {
+        textDecoration: 'none',
+      },
+      '&:active': {
+        color: vars.colors.$negativeContrast,
+      },
+      '&:visited': {
+        color: '#C870FF',
```

Comment:
We shouldn't be using strings like this for colors. If we need a new color it should be discussed with @isaozler if we can use an existing color, or to discuss how this fits into the design system